react/features/base/i18n/middleware.js

```diff
@@ -46,3 +52,27 @@ function _setConfig({ getState }, next, action) {
 
     return result;
 }
+
+/**
+ * Notifies the feature base/i18n that the action {@link APP_WILL_MOUNT} is
+ * being dispatched within a specific redux {@code store}.
+ *
+ * @param {Store} store - The redux store in which the specified {@code action}
+ * is being dispatched.
+ * @param {Dispatch} next - The redux {@code dispatch} function to dispatch the
+ * specified {@code action} to the specified {@code store}.
+ * @param {Action} action - The redux action {@code CONNECTION_ESTABLISHED}
+ * which is being dispatched in the specified {@code store}.
+ * @private
+ * @returns {Object} The new state that is the result of the reduction of the
+ * specified {@code action}.
+ */
+function _appWillMount(store, next, action) {
+    const result = next(action);
+
+    // setting the date formatter locale globally for the app.
+    // FIXME: languageDetector seems to be off and doesn't return anything ATM.
```

Comment:
you should be able to remove the fixme now, I expect the PR fixing that to land before this one

Reply:
Removed